docs/source/guides/contributor/chapters/ai_policy.rst

```diff
@@ -0,0 +1,121 @@
+            Signed-off-by:  <- Signature and acknowledgment of licensing terms when
+                            contributing to the project (created by git commit -s)
+
+-   **Originality:** If AI was used as a brainstorming or refactoring
+    tool, but the final code is substantially original and reviewed by
+    the contributor, a general disclosure in the PR description is
+    sufficient.
+
+Licensing and Copyright
+-----------------------
+
+Contributors are responsible for ensuring that AI-generated code complies
+with the Avocado's license (GPLv2).
+
+-   **Be Alert to Possible Copyright Issues:** Do not assume that
+    AI-generated code is free of licensing obligations or copyright.
+    Treat it with the same diligence as code copied from other sources.
+    To the extent practical, ensure that AI-generated code does not
+    infringe third-party copyrights.
+-   **Review for Training Data Leakage:** Contributors are expected to
+    enable any code similarity matching or blocking functionality offered
+    by their AI tools if it is available. This helps to identify or
+    prevent potential leakage from proprietary or incompatibly licensed
+    training data. If such instances are identified by the contributor,
+    the contributor must either (a) ensure that the AI-generated code is
+    used in a manner that complies with its license, and ensure that such
+    license is compatible with the license of the project, or (b) the
+    code should be re-written before submission.
+
+Code Quality, Review, and Testing
+----------------------------------
+
+AI-generated code must adhere to the same quality standards as
+human-written code.
+
+-   **Human Review Required:** All AI-generated code, regardless of its
+    source, **must** be thoroughly reviewed by a human contributor. This
+    review should be as rigorous, if not more so, than the review of
+    human-written code.
+-   **Testing:** Comprehensive tests (unit, integration, etc.) must be
+    written for AI-generated code, just as for any other code. This is
+    crucial for verifying its behaviour and preventing regressions.
+-   **Readability and Maintainability:** Ensure AI-generated code must
+    adhere to Avocado’s :ref:`commit_style_guide`
+    and pass the static-checks.
+
+AI contribution best practise
+-----------------------------
+
+-   **AI as an Assistant:** AI tools are encouraged as assistants for
+    boilerplate generation, refactoring suggestions, debugging, or
+    exploring different approaches. They should not replace critical
+    thinking, understanding, and human review.
+-   **Understanding is Key:** Contributors must fully understand any
+    AI-generated code they submit. Do not submit code you do not
+    comprehend or cannot debug yourself.
+-   **Iterative Refinement:** Use AI tools iteratively. Generate, review,
+    refine, and test. Do not blindly accept AI output.
```

Comment:
Again, generic usage of AI, people should know that when they use AI

Reply:
Yes, they should know, but I think we can't assume it, therefore I think we need to put this statement here.

Reply:
I agree with @ldoktor that this sounds like instructing people how to use AI tools. The only relevant part that still seems relevant is "Do not blindly accept AI output". But this is alreaady mentioned in the previous point where we phrase it better as "Do not submit code you do not comprehend or cannot debug yourself."

Reply:
Such a context pertains to a specific AI tools workflow and people could use AI tools in different ways. There is nothing wrong with the shortness of the "Do not blindly accept AI output" sentence and in fact it even sticks better to memory if it is brief like this and to the point.

Reply:
@clebergnu, @pevogam So IIUIC you propose to keep only `Do not blindly accept AI output` sentence here? I am ok with that.